policy/policy-authority_test.go

```diff
@@ -107,6 +107,16 @@ func TestWillingToIssue(t *testing.T) {
 		`www.google.com`,
 		`lots.of.labels.pornhub.com`,
 	}
+	blacklistContents := []string{
```

Comment:
I know this is from before your change but I don't really think we should be using real entities in our tests where possible.